src/components/Shared/SpectralSettings/SpectralSettingsComponent.tsx

```diff
@@ -53,7 +54,7 @@ export class SpectralSettingsComponent extends React.Component<{
                     </FormGroup>
                 )}
 
-                <FormGroup label={"System"} inline={true} disabled={disableSystemSetting}>
+                <FormGroup label={this.props.label ? " " : "System"} inline={true} disabled={disableSystemSetting}>
```

Comment:
I think we can keep the "System" label here to have consistent design with other widgets.

Reply:
modified